arch/x86_64/mcount.S

```diff
@@ -1,8 +1,54 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = %rbp */
+/* stack frame (with -pg): parent addr = 8(%rbp), child addr = 56(%rsp) */
```

Comment:
Ditto.  Please remove 56 here.  Also it'd be great if you could mention why fentry needs 48 and mcount needs 56 (regarding #92).  It's because of the 16-byte stack alignment required from GCC 7..